story3.js

```diff
@@ -0,0 +1,73 @@
+fetch('./films.json')
+    .then((response) => response.json())
+    .then((json) => {
+        let  films = json.films
```

Comment:
Split everything inside the `.then` block into one or more functions that you can call from here